README.md

```diff
@@ -390,6 +390,7 @@ If data is being replicated from S3 to S3 then Circus Train will use the AWS S3
 |`copier-options.canned-acl`|No|AWS Canned ACL name. See [Access Control List (ACL) Overview](https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl) for possible values. If not specified `S3S3Copier` will not specify any canned ACL.|
 |`copier-options.copier-factory-class`|No|Controls which copier is used for replication if provided.|
 |`copier-options.s3s3-retry-max-copy-attempts`|No|Controls the maximum number of attempts if AWS throws an error during copy. Default value is 3.|
+| `copier-options.assume-role`|No|ARN of an IAM role to assume when writing S3 data to the target S3 location. Useful when the target is in a different AWS account than CircusTrain is running in. Note that if JCEKS is also configured, JCEKS credentials will be used instead of assuming a role. If `assume-role` is not specified, the copier will use instance credentials. The role provided must have read access to the S3 source and write access to the S3 target.|
```

Comment:
CircusTrain -> Circus Train

Reply:
Just seen that its written the same for the other assume role config, perhaps we should change both? Or just leave it. I don't mind.

Reply:
I'll change any occurrences of 'CircusTrain' to have a space so its consistent with the rest of the readme